benchmarks/oscar/ootb/run_oscar_train.sh

```diff
@@ -0,0 +1,63 @@
+export HSA_ENABLE_SDMA=0
+
+NODES=${1:-1}
+GPUS=${2:-8}
+BS=${3:-128}
```

Comment:
We were previously using batch_size=32

Reply:
Here we pick a batch size as large as possible for A100. And for 8xA100 BS=256, for 16xMI250X it is 128. I will change the default value to 256. cc: @peixuanzuo